### DISCUSSION TITLE
Extended config with `:semanticPrefixFix` creates chore commit

### DISCUSSION THREAD

**Discussion by jdbruijn** [renovatebot/renovate#8661] [Request Help] [answered]
I've got my [default configuration](https://github.com/vidavidorra/.github/blob/main/default.json) on GitHub. The configuration in [vidavidorra/repo-template](https://github.com/vidavidorra/repo-template) extends that configuration as shown below. Today [PR #185](https://github.com/vidavidorra/repo-template/pull/185) was created that used a `chore` commit type. 

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["github>vidavidorra/.github", ":semanticPrefixFix"]
}
```

Is this a bug or is my configuration wrong? I'd expected [`:semanticPrefixFix`](https://docs.renovatebot.com/presets-default/#semanticprefixfix) to set the commit type to `fix`, but do I need to use [`:semanticCommitTypeAll`](https://docs.renovatebot.com/presets-default/#semanticcommittypeallltarg0gt) instead or something? If so, can someone please tell me what the difference between those two is?

**Comment by rarkins** [comment, marked as answer]
Using `:semanticPrefixFix` would work if it weren't for the fact that you're extending [config:base](https://docs.renovatebot.com/presets-config/#configbase) which includes [:semanticPrefixFixDepsChoreOthers](https://docs.renovatebot.com/presets-default/#semanticprefixfixdepschoreothers).

So you could:
- Use `:semanticCommitTypeAll` instead
- Add `:semanticPrefixFixDepsChoreOthers` to [ignorePresets](https://docs.renovatebot.com/configuration-options/#ignorepresets)
- Instead of extending `config:base`, remove it and add only the presets it contains that you want

**Reply by jdbruijn** [reply]
Thanks for the quick reply, I'll try that and mark as answer if it indeed works.

My understanding with the config, not sure where I got this from, was that the last config would overwrite the previous ones. Is that wrong then?

**Reply by jdbruijn** [reply]
> * Add `:semanticPrefixFixDepsChoreOthers` to [ignorePresets](https://docs.renovatebot.com/configuration-options/#ignorepresets)

Worked a treat ([proof](https://github.com/vidavidorra/repo-template/pull/185)), thanks again!

**Reply by rarkins** [reply]
> My understanding with the config, not sure where I got this from, was that the last config would overwrite the previous ones. Is that wrong then?

packageRules are executed in order, after base config. So because the one you were trying to override was a package rule, it was executed after. It's not perfect/intentional - we're trying to work out a better/more predictable way for a future breaking release.
